components/camel-milo/src/test/java/org/apache/camel/component/milo/MonitorItemMultiConnectionsCertTest.java

```diff
@@ -39,6 +41,8 @@
 /**
  * Testing monitoring items over multiple connections
  */
+// Disabled due to Milo 1.0.5 API migration - certificate-based authentication needs further investigation
+@Disabled
```

Comment:
the reason can be put in the annotation directly.
Advantages:
- allows to see it in reports when tests are played
- less chance to have an outdated comment remaining when the disbaled annotation is removed

```
@Disabled("due to Milo 1.0.5 API migration - certificate-based authentication needs further investigation")